src/test/scala/chiselTests/DontTouchSpec.scala

```diff
@@ -123,4 +123,66 @@ class DontTouchSpec extends ChiselFlatSpec with Utils {
     // Ensure can compile the result.
     compile(new HasProbesAndProperties())
   }
+
+  "dontTouch.modulePorts" should "mark all ports of a module" in {
+    class ModuleWithPorts extends Module {
```

Comment:
can we add a test for a Module with property and/or Probe ports? Im unclear if those are expected to be annotated, or there is supposed to be an error, or...?